docs/relying-parties/how-twos/apple-iap.md

```diff
@@ -0,0 +1,105 @@
+
+Importantly, Apple provides read-only access to Apple IAP subscriptions. Consequently, we do not (and cannot) modify Apple IAP subscriptions (e.g. making plan changes such as upgrades).
+
+A more technical discussion (geared toward FxA engineers) can be found in [Apple IAP Integration](/ecosystem-platform/tutorials/subscription-platform#apple-iap-integration).
+
+## Prerequisites
+This guide assumes you are already integrated with FxA/the Subscription Platform; please see [Integration with Subscription Platform](/ecosystem-platform/relying-parties/tutorials/integration-with-subscription-platform) for more information.
+
+Ideally, you will also want to grant 1-2 Subscription Platform engineers working on the integration developer access to your iOS app in [App Store Connect](https://appstoreconnect.apple.com/) ([example bug](https://bugzilla.mozilla.org/show_bug.cgi?id=1760053)). This allows us to access your iOS app’s [App Store Server API](https://developer.apple.com/documentation/appstoreserverapi) keys, to look up needed app-specific information and to configure an endpoint to receive App Store Server notifications.
+
+## Configuration
+These steps can be done by a product manager or developer and in parallel with steps in [SubPlat API Calls](#subplat-api-calls). More information regarding specific Stripe metadata keys mentioned below can be found in the [Subscription Platform integration tutorial](/ecosystem-platform/tutorials/subscription-platform#stripe-productplans).
+
+### Create an IAP configuration document
+This document represents a list of App Store plans (identified uniquely by their App Store [`productId`s](https://developer.apple.com/documentation/appstoreserverapi/productid/) for your product.
+
+1. Go into App Store Connect and find your iOS app’s human-readable [`productId`](https://developer.apple.com/documentation/appstoreserverapi/productid/)s and [`bundleId`](https://developer.apple.com/documentation/appstoreserverapi/bundleid/).
+2. Create a configuration document similar in form to https://api.accounts.firefox.com/v1/oauth/subscriptions/iap/plans/guardian-vpn (omitting Android plans) including a `bundleId` property. This can be in a Google document for now.
+    * Note: The only information SubPlat uses from the VPN document currently is the `productId` and `platform`. The VPN configuration document is structured in this way for historical reasons. We may change how this information is structured for future integrations.
+3. File a ticket in the [FXA Jira project](https://mozilla-hub.atlassian.net/browse/FXA) to create a new document in SubPlat's "IAP config" Firestore collection, and provide the document from Step 2 along with an `appName`.
+    * `appName` is a human-readable name that SubPlat will use internally to uniquely identify your app in our system (e.g. `"guardian-vpn"` is the `appName` for the Mozilla VPN in the link above).
```

Comment:
This is used in the URL for the endpoint to register with, so its not purely internal use.